packages/neon-api/src/provider/common.ts

```diff
@@ -61,3 +61,21 @@ export function findGoodNodesFromHeight(
   const threshold = bestHeight - tolerance;
   return sortedNodes.filter((n) => n.height >= threshold);
 }
+
+export interface Entry {
+  txid: string;
+  time: number;
+  block_height: number;
+  asset: string;
+  amount: string;
+  address_to: string;
+  address_from: string;
+}
+
+export interface AddressAbstract {
+  total_pages: number;

```

Comment:
All the fields in the internal/common interfaces should use camelcase for consistency. Please adhere to the convention.